CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+1.4.0 / 2019-08-28
+==================
+
+###### Changes
+* Dropping support for PHP 5.3
+* Updated documentation about testing
+* Syn library is [0.0.2](https://github.com/bitovi/syn/tree/v0.0.2)
+* `setDesiredCapabilities` will throw an exception if you will try to set capabilities on already started session
+* Added `getDesiredCapabilities` to allow fetch capabilities
```

Comment:
Please move into `New features` sub-section and replace with:

```
* Added `getDesiredCapabilities` method for fetching current desired capabilities
```

Reply:
moved.